sensor_core/dsp/dsp_manager.py

```diff
@@ -1,3 +1,15 @@
 import numpy as np
+import scipy.signal as signal
 
-#TODO: add modules for 1D dsp 
\ No newline at end of file
+#Moving average filter
+def moving_avg_filter(data, N):
+    data_arr = np.concatenate(([np.min(data)] * (N), data))
+    cumsum = np.cumsum(data_arr)
+    yi = (cumsum[N:] - cumsum[:-N]) / float(N)
```

Comment:
Here the output data is 'yi', in butterworth filter function the output is filtered. Should keep the output variable consistent

Reply:
Any particular reason to typecast N as a float?